enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchAcceptanceTest.scala

```diff
@@ -42,11 +42,11 @@ class MatchAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsTe
 
   test("really weird query that breaks IDP") {
     val result = executeWithAllPlannersAndCompatibilityMode(
-      """MATCH (a)
-        |WITH a WHERE true
-        |MATCH (c), (a)-[r]->(x)
-        |WHERE (a)-->(c)
-        |RETURN *""".stripMargin)
+    """MATCH (a)
+      |WITH a WHERE true
+      |MATCH (c), (a)-[r]->(x)
+      |WHERE a.foo = c.bar
+      |RETURN *""".stripMargin)
 
     result.size should equal(0)// does not throw
```

Comment:
Is it a lot of extra work to make the test return at least one row and then assert on the result? Maybe add to MatchAcceptance.feature instead ( or maybe it is not there in 3.2?).